pkg/network/ebpf/c/protocols/classification/protocol-classification.h

```diff
@@ -193,6 +193,13 @@ __maybe_unused static __always_inline void protocol_classifier_entrypoint(struct
             }
         }
         return;
+    } else {
```

Comment:
also, now it seems that we're going to set PLAINTEXT tag for every connection, even if it does not have supported protocols.
I'm concerned about the impact of the map utilization and it can lead into stuffing the map with "irrelevant" entries and then we won't have enough space for the relevant entries